apollo-gradle-plugin/src/test/testProject/android/buildScriptFixtures/basic.gradle

```diff
@@ -37,6 +44,9 @@ repositories {
 
 dependencies {
   compile 'com.android.support:appcompat-v7:25.1.1'
+  compile 'com.google.code.findbugs:jsr305:3.0.1'
+  compile 'javax.annotation:jsr250-api:1.0'
+  compile 'com.fernandocejas:arrow:1.0.0'
```

Comment:
I think we can use Date or any other Custom type to not carry this additional dependency....